dashboard/app/views/reports/csp_pd_responses.html.haml

```diff
@@ -2,11 +2,17 @@
   CS Principles PD Responses
 
 - if @responses.size > 0
+  #{link_to 'Download CSV', {:format => :csv}}
+
   - @responses.each do |level_id, level_responses|
     %h3
       CSP PD Responses for Level
       = level_id
     - if level_responses.size > 0
+      - if level_responses.size == @response_limit
+        WARNING: The results are truncated, there are additional records not
```

Comment:
Yep, I realized that when writing this. Is it worth it to add an extra variable in the controller indicating whether results were truncated and displaying based on that? Happy to do so, I decided it didn't matter enough. Note that, depending on how much this gets used, pagination may eventually get implemented.